webservice/utils/get-data-from-form-section.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var assign = require('es5-ext/object/assign');
+
+module.exports = function getDataFromSection(dataForm, result, typeMapper) {
+
+	if (!result) result = {};
+
+	var ownerType = dataForm.propertyMaster;
+
+	// first try toWsSchemaJson method
+	if (dataForm.toWebServiceSchemaJSON) {
+		var dataFormResult = dataForm.toWebServiceSchemaJSON(typeMapper);
+		return assign(result, dataFormResult);
+	}
+
+	// if tabular section
+	if (dataForm.propertyName) {
+
+		// exclude customs
+		if (!ownerType.get(dataForm.propertyName)
```

Comment:
What you really want to do here is check if `!ownerType.get(dataForm.propertyName) instanceof db.NestedMap` and it is enough. First of all `ownerType.get(dataForm.propertyName)` is true for proper configuration of section. Secondly `getItemType` is just duck typing for `NestedMap`.